doc/classes/ProceduralSky.xml

```diff
@@ -12,36 +15,52 @@
 	</methods>
 	<members>
 		<member name="ground_bottom_color" type="Color" setter="set_ground_bottom_color" getter="get_ground_bottom_color">
+			Color of the ground at the bottom
 		</member>
 		<member name="ground_curve" type="float" setter="set_ground_curve" getter="get_ground_curve">
+			How quickly the [member ground_horizon_color] fades into the [member ground_bottom_color].
 		</member>
 		<member name="ground_energy" type="float" setter="set_ground_energy" getter="get_ground_energy">
+			Amount of energy contribution from the ground.
 		</member>
 		<member name="ground_horizon_color" type="Color" setter="set_ground_horizon_color" getter="get_ground_horizon_color">
+			Color of the ground at the horizon.
 		</member>
 		<member name="sky_curve" type="float" setter="set_sky_curve" getter="get_sky_curve">
+			How quickly the [member sky_horizon_color] fades into the [member sky_top_color].
 		</member>
 		<member name="sky_energy" type="float" setter="set_sky_energy" getter="get_sky_energy">
+			Amount of energy contribution from the sky.
 		</member>
 		<member name="sky_horizon_color" type="Color" setter="set_sky_horizon_color" getter="get_sky_horizon_color">
+			Color of the sky at the horizon.
 		</member>
 		<member name="sky_top_color" type="Color" setter="set_sky_top_color" getter="get_sky_top_color">
+			Color of the sky at the top.
 		</member>
 		<member name="sun_angle_max" type="float" setter="set_sun_angle_max" getter="get_sun_angle_max">
+			Distance from center of sun where it fades out completely.
 		</member>
 		<member name="sun_angle_min" type="float" setter="set_sun_angle_min" getter="get_sun_angle_min">
+			Distance from sun where it goes from solid to starting to fade.
 		</member>
 		<member name="sun_color" type="Color" setter="set_sun_color" getter="get_sun_color">
+			Color of the sun.
 		</member>
 		<member name="sun_curve" type="float" setter="set_sun_curve" getter="get_sun_curve">
+			How quickly the sun fades away between [member sun_angle_min] and [member sun_angle_max]
 		</member>
 		<member name="sun_energy" type="float" setter="set_sun_energy" getter="get_sun_energy">
+			Amount of energy contribution from the sun.
 		</member>
 		<member name="sun_latitude" type="float" setter="set_sun_latitude" getter="get_sun_latitude">
+			Angle of the sun in the sky vertically. 
```

Comment:
Vertical angle of the sun? Not sure.

Reply:
What about 'vertical position of the sun described as an angle above the horizon'?